tests/test_cape_result.py

```diff
@@ -31835,6 +31873,56 @@ def test_process_buffers(process_map, correct_buffer_body, correct_tags, correct
                 for value in values:
                     buffer_ioc_table.add_tag(tag, value)
             assert check_section_equality(parent_section.subsections[0], correct_result_section)
+        should_have_network_buffer = False
```

Comment:
the following logic is way too involved for a unit test. instead you should just include values in the `parametrize` declaration for expected results for given inputs